tests/src/test/scala/it/unibo/scafi/test/functional/stdlib/TimeUtils.scala

```diff
@@ -0,0 +1,65 @@
+package it.unibo.scafi.test.functional.stdlib
+
+import it.unibo.scafi.test.FunctionalTestIncarnation._
+import it.unibo.scafi.test.functional.ScafiAssertions.assertNetworkValues
+import it.unibo.scafi.test.functional.ScafiTestUtils
+import it.unibo.utils.StatisticsUtils.stdDev
+import org.scalatest._
+
+import scala.concurrent.duration._
+
+class TimeUtils extends FlatSpec{
+  import ScafiTestUtils._
+
+  val Time_Utils = new ItWord
+
+  private[this] trait SimulationContextFixture {
+    val net: Network with SimulatorOps = standardNetwork()
+  }
+
+  private[this] trait TestProgram extends AggregateProgram with StandardSensors with BuildingBlocks
+
```

Comment:
The following test cases are ok, but they only test the stabilised field.
Could you also test intermediate states, e.g., by verifying that executing a number of rounds by which no floor should be hit, values lies within an interval. For instance, consider `ceiling=10` and `floor=0`; if we run `18+5` rounds in a round-robin fashion (to do so, the simulator should be extended to receive a "scheduling policy"), all values should be `8` or `7`.